packages/socket/lib/utils.ts

```diff
@@ -0,0 +1,39 @@
+import * as socketIoParser from 'socket.io-parser'
+// @ts-ignore
+import * as engineParser from 'engine.io-parser'
+
+export const encode = (data: any, namespace: string) => {
+  return new Promise((resolve) => {
+    const encoder = new socketIoParser.Encoder()
+    const socketIoEncodedData = encoder.encode({
+      type: socketIoParser.PacketType.EVENT,
+      data,
+      nsp: namespace,
+    })
+
+    engineParser.encodePayload(socketIoEncodedData.map((item) => {
+      return {
+        type: 'message',
+        data: item,
+      }
+    }), (encoded: any) => {
+      resolve(encoded)
+    })
+  })
+}
+
+export const decode = (data: any) => {
```

Comment:
Type the return type

Reply:
Updated: [`4e1085d` (#27592)](https://github.com/cypress-io/cypress/pull/27592/commits/4e1085dc7cbee1be2225ce93bccbcac79deafe8a)